src/components/home/home.scss

```diff
@@ -14,7 +17,7 @@ $bg-img: url('../../images/meal-prep.jpg');
     }
```

Comment:
This .cta-block is different in the mockup for the bigger screens. The layers are laid on top of each other like this:
first, the hero block with the background image of the meals.
second, the white, transparent overlay background of the .cta block.